lib/mixlib/install/backend/artifactory.rb

```diff
@@ -54,24 +56,31 @@ def info
           end
         end
 
-        private
+        def product_versions
+          params = {
+            g: "com.getchef",
+            a: options.product_name,
+            repos: ARTIFACTORY_REPOSITORY
+          }
+          results = client.get("/api/search/versions", params)["results"]
```

Comment:
Sure. I just don't want us wasting time on why GAVC logic is not sorting correctly. FWIW the Artifactory team is mostly ignoring GAVC based queries and encouraging heavy use of AQL and property based searches. Pretty sure the newer versions of Artifactory are being optimize for this approach.